gems/sorbet-runtime/test/types/props/_props.rb

```diff
@@ -283,26 +287,24 @@ class TestCustomProps
     end
 
     it 'work when plain' do
-      @obj.custom = "foo"
-      assert_equal("foo", @obj.custom)
+      @obj.custom = MyCustomType.new("foo")
+      assert_equal("foo", @obj.custom.value)
     end
 
     it 'work when nilable' do
       @obj.nilable_custom = nil
       assert_nil(@obj.nilable_custom)
 
-      @obj.nilable_custom = "foo"
-      assert_equal("foo", @obj.nilable_custom)
+      @obj.nilable_custom = MyCustomType.new("foo")
+      assert_equal("foo", @obj.nilable_custom.value)
     end
 
     it 'work as collection element' do
-      skip "this isn't supported"
-
```

Comment:
Nice! Now there's only one skip left in `sorbet-runtime`, this one:

https://github.com/sorbet/sorbet/blob/1dad9dc018bacb02c0395a8b5b2d923799207804/gems/sorbet-runtime/test/types/props/decorator.rb#L90-L97